moped-editor/src/views/dashboard/DashboardStatusModal.js

```diff
@@ -31,7 +31,7 @@ const DashboardStatusModal = ({
     <>
       <Typography
         component={"span"}
-        className={classes.fieldLabelText}
+        className={modalParent === "summary" && classes.fieldLabelText}
```

Comment:
The production version of the Dashboard's status modal has [some styles](https://github.com/cityofaustin/atd-moped/blob/0041b4e897a4bacb53d172570de04a110678a1f5/moped-editor/src/views/dashboard/DashboardStatusModal.js#L22-L27) that apply the pointer cursor and shrink the (+) icon a bit. can you add those back?

also, the linter is barking b/c `false` ends up getting passed as a className:
```
Failed prop type: Invalid prop `className` of type `boolean` supplied to `ForwardRef(Typography)`, expected `string`.
```

Reply:
🐶 ill add them back now